incubator/efs-provisioner/templates/clusterrole.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.rbac.create }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "efs-provisioner.fullname" . }}
+  labels:
+    app: {{ template "efs-provisioner.fullname" . }}
```

Comment:
The `app` label should be `{{ template "efs-provisioner.name" . }}`. Apply throughout the chart.